charts/model-engine/templates/service_template_config_map.yaml

```diff
@@ -95,14 +95,17 @@ data:
             {{- toYaml . | nindent 12 }}
           {{- end }}
           {{- if eq $device "gpu" }}
-          {{- if empty $node_selector }}
-          nodeSelector:
-          {{- end }}
-            k8s.amazonaws.com/accelerator: ${GPU_TYPE}
+          # {{- if empty $node_selector }}
+          # nodeSelector:
+          # {{- end }}
+          #   k8s.amazonaws.com/accelerator: ${GPU_TYPE}
```

Comment:
will re-add this, just serves as a note that need to add documentation that this tag must be set on nodegroups